pkg/interfaces/contracts/vault/IAggregatorRouter.sol

```diff
@@ -28,6 +31,31 @@ interface IAggregatorRouter {
         bytes calldata userData
     ) external returns (uint256 amountOut);
 
+    /**
+     * @notice Executes a swap operation specifying an exact input token amount.
+     * @dev Differently from `swapSingleTokenExactIn`, the sender should transfer the maxAmountIn to the Vault before
+     * calling this function, and the router will transfer any leftovers back to the sender after the swap is
+     * calculated.
```

Comment:
```suggestion
     * @dev The sender should transfer the maxAmountIn to the Vault before calling this function, and the router will
     * transfer any leftovers back to the sender after the swap is calculated.
```
Differently from isn't standard usage; I don't think we need it to contrast this with exact in.